apps/test/unit/templates/AgeDialogTest.js

```diff
@@ -1,28 +1,40 @@
-import {assert} from '../../util/deprecatedChai';
+import {assert} from '../../util/reconfiguredChai';
 import React from 'react';
 import {UnconnectedAgeDialog as AgeDialog} from '@cdo/apps/templates/AgeDialog';
 import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {replaceOnWindow, restoreOnWindow} from '../../util/testUtils';
 
 describe('AgeDialog', () => {
   const defaultProps = {
     signedIn: false,
     turnOffFilter: () => {}
   };
 
-  afterEach(() => sessionStorage.clear());
+  before(() => {
+    replaceOnWindow('sessionStorage', {
+      getItem: () => {},
+      setItem: () => {}
+    });
+  });
+
+  after(() => {
+    restoreOnWindow('sessionStorage');
+  });
 
-  it('renders null if signed in', () => {
+  it('renders null if user is signed in', () => {
     const wrapper = shallow(<AgeDialog {...defaultProps} signedIn={true} />);
     assert.equal(wrapper.children().length, 0);
   });
 
-  it('renders null if seen before', () => {
-    sessionStorage.setItem('ad_anon_over13', true);
+  it('renders null if dialog was seen before', () => {
+    let getItem = sinon.stub(window.sessionStorage, 'getItem').returns('true');
```

Comment:
Could avoid weakening the test's assertion here by only stubbing `getItem` with the particular named argument:
```suggestion
    let getItem = sinon.stub(window.sessionStorage, 'getItem').withArgs('ad_anon_over13').returns('true');
```